tt_eager/tt_dnn/op_library/composite/composite_ops.cpp

```diff
@@ -237,11 +237,17 @@ Tensor multigammaln(const Tensor& a, const MemoryConfig& output_mem_config) {
 // use transformation y = x*tanh[softplus[x]] by broadcast
 // Ref: https://krutikabapat.github.io/Swish-Vs-Mish-Latest-Activation-Functions/
 Tensor _mish(const Tensor& x, const MemoryConfig& output_mem_config) {
-    Tensor sp_x = softplus(x, 1.0f, 20.0f, output_mem_config);
-    Tensor tanh_x = tanh(sp_x, output_mem_config);
-    sp_x.deallocate();
-    Tensor mish_x = mul(x, tanh_x, std::nullopt, output_mem_config);
-    return mish_x;
+    std::vector<Tensor> output_tensors = {Tensor(operation::get_workers_for_op_output({x}))};
+    operation::launch_op(
+        [output_mem_config] (const std::vector<Tensor>& input_tensors, const std::vector<std::optional<const Tensor>>& optional_input_tensors) mutable -> std::vector<Tensor> {
+            const auto& x = input_tensors.at(0);
+            Tensor sp_x = softplus(x, 1.0f, 20.0f, output_mem_config);
+            Tensor tanh_x = tanh(sp_x, output_mem_config);
```

Comment:
neat, so this `tanh_x` tensor gets allocated and destructed entirely in each worker thread?

Reply:
yessir

Reply:
zero extra sync overhead, nice